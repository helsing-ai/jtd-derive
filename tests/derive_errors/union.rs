#[derive(jtd_derive::JsonTypedef)]
union Foo {
    bar: u32,
}

fn main() {}
