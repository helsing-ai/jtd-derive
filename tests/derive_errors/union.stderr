error: jtd-derive does not support unions
 --> tests/derive_errors/union.rs:2:7
  |
2 | union Foo {
  |       ^^^
